frontend/src/utils/toasts/errorToast.js

```diff
@@ -0,0 +1,23 @@
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const errorToast = data => {
+  let error;
+
+  if (data.errors.hasOwnProperty("title")) {
+    error = `Title ${data.errors.title}.`;
+  } else {
+    error = "Something went wrong. Please try again later.";
+  }
+
+  toast.error(error, {
+    position: "top-right",
+    autoClose: 6000,
```

Comment:
6s??? isn't it too long???

Reply:
I've checked and it does feel a bit too long ;)